system-helper/org.freedesktop.Flatpak.rules.in

```diff
@@ -1,12 +1,14 @@
 polkit.addRule(function(action, subject) {
-    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
-         action.id == "org.freedesktop.Flatpak.runtime-install"||
+    if ((action.id == "org.freedesktop.Flatpak.runtime-install"||
```

Comment:
I realize this change is not new with this rebase, but wouldn't it make sense to also allow runtime installs? If the goal is for non-admins to be able to install apps.

Reply:
Maybe, but I don't know the history of it. I just reapplied the change. The original discussion is in https://phabricator.endlessm.com/T27854#757523 and https://github.com/endlessm/flatpak/pull/185 and https://github.com/endlessm/flatpak/pull/187. I'd prefer to keep this as is and open a separate discussion for it if the behavior isn't right.

Reply:
Never mind, parental controls are only checked  for apps not runtimes, so that's the reason we special case `app-install`.